CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 <img align="right" width="150" height="150" top="100" src="./assets/logo_circle.svg">
 
-# Contributing to the Inverter Network Contracts
+# Contribution Guidelines
```

Comment:
Do you think it is a good idea to add a link to  the testing guidelines somewhere? Maybe not rn but soonish

Reply:
Yeah, once we polished that doc, I think that's a good idea. Probably fine if we add that within the next 2-3 weeks.a